code/game/jobs/job/civilian.dm

```diff
@@ -145,6 +145,7 @@
 	idtype = /obj/item/weapon/card/id/cargo
 	access = list(access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_mining, access_mining_station)
 	minimal_access = list(access_maint_tunnels, access_cargo, access_cargo_bot, access_mailsorting)
+	alt_titles = list("Cargo Courier")

```

Comment:
You missed this one in your list, or I did.

Reply:
I must have missed it in my list, lemmie mark it.